Breakout functions (#47)
datacarpentry/astronomy-python: _episodes/05-join.md

> 3. We'll write the results to a file for use in the next notebook.
{: .checklist}

## Starting from this episode

> In the previous episode, we define a rectangle around stars in GD1 in spatial coordinates

Reviewer: ```suggestion
In the previous episode, we define a rectangle around stars in GD-1 in spatial coordinates
```

and in proper motion which we transformed into ICRS coordinates and created point
lists of the polygon verticies.
We will use that data for this episode. 
Whether you are working from a new notebook or coming back from a checkpoint, 
reloading the data will save you from having to run the query again. 

If you are starting this episode here or starting this episode in a new notebook,
you will need run the following lines of code:

This imports previously imported functions:
~~~
from astroquery.gaia import Gaia
import pandas as pd

from episode_functions import *
~~~
{: .language-python}

This loads in the data (instructions for downloading data can be

> found in the [setup directions](../setup.md))

Reviewer: ```suggestion
found in the [setup instructions](../setup.md))
```

~~~
{: .language-python}

> Now we can assemble the query the sky and proper motion point lists we compiled in episode 4.

Reviewer: ```suggestion
Now we can assemble the query using the sky and proper motion point lists we compiled in episode 4.
```
